File: 250517/남녀노소 구분짓기/sex-and-age.js
const fs = require("fs");

let input = fs.readFileSync(0).toString().trim().split("\n").map(Number);

let [gender, age] = input;

if (gender) {
    if (age >= 19) {
        console.log("WOMAN");
    }
    else {
        console.log("GIRL");
    }
}
else {
    if (age >= 19) {
        console.log("MAN");
    }
    else {
        console.log("BOY");
    }
}

File: 260312/남녀노소 구분짓기/sex-and-age.py
import sys

input = sys.stdin.readline

def multi_input(n: int):
    return [input() for _ in range(n)]

gender, age = map(int, multi_input(2))

if gender == 0:
    if age >= 19:
        print('MAN')
    else:
        print('BOY')
else:
    if age >= 19:
        print('WOMAN')
    else:
        print('GIRL')